src/app.ts

```diff
@@ -8,6 +8,7 @@ import { getDataDefByHash } from "./reverse_map"
 import { providers } from "./providers"
 import { app as api, CONFIG as apiConfig } from "./api"
 import { generate, makeConfig } from "./api/configurator"
+import { IDataProvider } from "./IDataProvider"
 
 console.log("hello")
```

Comment:
is it really needed here?

Reply:
This log line indicates that the app started launching. Maybe wording (`hello`) may be better, e.g. `starting app`. This PR doesn't make any changes to this behavior, let's discuss it in another issue.